hbase-server/src/main/java/org/apache/hadoop/hbase/monitoring/MonitoredRPCHandlerImpl.java

```diff
@@ -43,6 +43,8 @@
   private String methodName = "";
   private Object [] params = {};
   private Message packet;
+  private boolean isSnapshot = false;
```

Comment:
fyi, isSnapshot is a method name, not a datamember name. s/isSnapshot/snapshot/?

Reply:
This is a nit. Can fix later.

Reply:
Thanks, I'll change it.